src/ZkFold/Base/Protocol/ARK/Plonk.hs

```diff
@@ -45,9 +46,11 @@ data Plonk (d :: Natural) (n :: Natural) t = Plonk F F F (Vector n Natural) (Ari
 -- TODO (Issue #25): make a proper implementation of Arbitrary
 instance Arbitrary (Plonk d n t) where
     arbitrary = do
-        let (omega, k1, k2) = getParams 5
+        nP <- integerToNatural <$> chooseInteger (1, 10)
+        let (omega, k1, k2) = getParams nP
         ac <- arbitrary
-        Plonk omega k1 k2 (Vector [1, 2]) ac <$> arbitrary
+        nV <- integerToNatural <$> chooseInteger (1, 100)
+        Plonk omega k1 k2 (Vector [1..nV]) ac <$> arbitrary
```

Comment:
The vector here gives the indices of public input variables. For now, it should coincide with the circuit inputs.

Note: We are adding private inputs soon, so in the future, this vector will only be a subset of the circuit inputs.